tss/common/base.go

```diff
@@ -56,6 +62,7 @@ func (b *BaseTss) ProcessInboundMessages(ctx context.Context, msgChan chan *comm
 		select {
 		case wMsg := <-msgChan:
 			{
+				span.AddEvent("Process inbound message", traceapi.WithAttributes(attribute.String("p2pmsg.from", wMsg.From.String()), attribute.String("p2pmsg.type", wMsg.MessageType.String())))
```

Comment:
Would be nice to also have the msg payload.
Same for outbound

Reply:
added